### ISSUE TITLE
UX: Split frontend bundle by language

### ISSUE THREAD

**Issue by benmccann** [photoprism/photoprism#821]
I noticed that https://demo.photoprism.org/static/build/app.js is 3.6MB, which is fairly large. I didn't build it locally, so was just looking at the minified version, but one thing I saw is that there's lots of non-English text, which presumably I shouldn't need. E.g. you can do a search for "Отключает одновременное преобразование" and find it in the bundle.

**Comment by lastzero** [comment]
You are right with this one as well 👍  Should be done, although it works for now.

**Comment by benmccann** [comment]
There's a `translations.json` file checked in that contains all languages. It would need to be split into multiple. It looks like `gettext-compile` does not support this (https://github.com/Polyconseil/easygettext/issues/99), so we may have to post-process the file ourselves

We might then be able to update `Vue.$translations`. It's unclear to me if updating this variable is something that `vue-gettext` supports, but I suppose it should. We'd have to make it loaded reactively based upon the language configuration value which can be updated in the settings screen

**Comment by lastzero** [comment]
This has been done along with a number of other optimizations such as lazy-loading MapLibre GL JS, which together have reduced the JS bundle file size by 54% to just 690 KB (compressed):

- https://github.com/photoprism/photoprism/issues/4778#issuecomment-2749392069

To further improve loading performance, it seems best to shrink the icon font next, rather than removing or lazy-loading additional JS packages. We are open to other suggestions as well! 🐇
